lona/html/parsing.py

```diff
@@ -128,6 +153,15 @@ def handle_starttag(self, tag, attrs):
 
             node_attributes[name] = value
 
+        # textareas
+        if self._node.tag_name == 'textarea':
+            self._node.value = self._node.value + self.render_start_tag(
```

Comment:
It seems there is an easier solution. Just add
```py
class NodeHTMLParser(HTMLParser):
    CDATA_CONTENT_ELEMENTS = HTMLParser.CDATA_CONTENT_ELEMENTS + ("textarea",)
```

Reply:
@maratori Oh! Thats much simpler indeed :D Thanks for researching this, this class attribute seems not to be documented

Fixed!